oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/PersistenceTimer.java

```diff
@@ -43,16 +48,18 @@
 @Slf4j
 public enum PersistenceTimer {
     INSTANCE;
-    private Boolean isStarted = false;
+    @VisibleForTesting
+    Boolean isStarted = false;
```

Comment:
You have `@VisibleForTesting`, why remove private? And why use `Boolean`, rather than `boolean`. I can't see anyone uses it as NULL. I know this is a legacy fault, but please fix it.